simpleviewinc/keg-cli
# Upgraded Expo-CLI

**Ticket**: [ZEN-229](https://jira.simpleviewtools.com/browse/ZEN-229)

## Context

* ZEN-229 tasked me to update our dependencies, and we should update expo-cli too

## Goal

* update expo-cli to the latest version

## Updates

* `containers/**/Dockerfile`
  * updated FROM directive for second build steps to use the node version defined in defaults.env
* `scripts/setup/defaults.env`
  * updated to include the node image version to be 12.13, which is needed for the latest expo-cli versions, and the expo-cli latest version
* `containers/base/Dockerfile`
  * updated to use the environment variable from `defaults.env` for expo-cli
* `scripts/**`
  * updated to include a script to update the cli's local configuration


## Testing
* `keg cli update`
* For each of the keg containers [ core, tap, components, regulator ]:
  * `keg <container> start`
  * verify that it runs/renders as expected
 * `keg <container> att`
 * verify that the output of `node --version` is `12.13.1`
 * verify that the output of `expo-cli --version` is `3.22.3`
